src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
```

Comment:
Consider using CSS from the previous task to make modifiers work as expected

Reply:
I don't understand what it means. I used block "stars" from the previous task, everything works, what do I need to change?

Reply:
https://github.com/mate-academy/layout_product-cards/assets/61904995/71e0a520-22c9-42bd-a4a4-4c0db58361dd